File: CH_05_03.py
FRIENDS = [
    {"name": "Graham"},
    {"name": "John"}, 
    {"name": "Terry"}, 
    {"name": "Eric"}, 
    {"name": "Michael"}
]

NAME_STRINGS = [friend["name"] for friend in FRIENDS]

NAMES_WITH_H = [string for string in NAME_STRINGS if "h" in string]

print("          =====NAME_STRINGS=====")
print(NAME_STRINGS)


print("          =====NAMES_WITH_H=====")
print(NAMES_WITH_H)

File: CH_05_03.js
const friends = [
    {name: "Graham"},
    {name: "John"}, 
    {name: "Terry"}, 
    {name: "Eric"}, 
    {name: "Michael"}
  ];
  
const nameStrings = friends.map(friend => friend.name);
const namesWithH = nameStrings.filter(name => name.includes("h"));

console.log("=====nameStrings=====");
console.log(nameStrings);

console.log("=====namesWithH=====");
console.log(namesWithH);
